Source/Core/DolphinNoGUI/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+if(NOT(USE_X11 OR ENABLE_HEADLESS))
+  return()
+endif()
+
+if(APPLE)
+  set(DOLPHIN_EXE_BASE Dolphin)
+else()
+  set(DOLPHIN_EXE_BASE dolphin-emu)
+endif()
+
+set(DOLPHIN_NOGUI_EXE ${DOLPHIN_EXE_BASE}-nogui)
```

Comment:
Maybe it could just be "dolphin-emu-nogui" on all platforms, we wouldn't need the variable and conditional then.